spec/features/pipelines/cherrypick/cherrypick_pipeline_spec.rb

```diff
@@ -22,30 +23,32 @@
   let!(:target_purpose) { create :plate_purpose }
 
   before do
-    assets = plates.each_with_object([]) do |plate, wells|
-      wells.concat(plate.wells)
-      plate.wells.each_with_index do |well, index|
-        well.well_attribute.update!(
-          measured_volume: 30 + (index % 30),
-          current_volume: 30 + (index % 30),
-          concentration: 205 + (index % 50)
-        )
+    step 'Before' do
```

Comment:
This looks like a lot of changes, but its mostly just grouping into steps for improved logging.